docs/_config.yml

```diff
@@ -34,13 +36,16 @@ t:
     advanced: Advanced concepts
     start: Getting started
     steps: Workflow steps
+    deploying: Deploying
```

Comment:
Side bar section is "Deploying" with the article names as "Deploy to XYZ." We can rename the section title to "Deploys" or "Deployments" or "Hosting" or anything else that may be better.

![image](https://user-images.githubusercontent.com/21328/94613753-ef981b00-0259-11eb-8b69-67fc781cad9d.png)

Reply:
"Deployment" is my preference for a header name since the guides are "Deploying"

Reply:
Makes sense to me. I've updated the header name to be "Deployments." I pluralized it to match the "Basic concepts," "Advanced concepts," and "Workflow steps" sections.

Fixed in f628553